### DISCUSSION TITLE
весь список команд и возможностей full list of commands and capabilities

### DISCUSSION THREAD

**Discussion by igorSof** [fedorovvl/tso_client#349] [Q&A]
Можно как-то ознакомиться со всем списком команд и возможностей?
Хочу написать скрипт, но не знаю даже с чего начать.
Какие есть команды, свойства, методы, что они делают что возвращают?

Is there a way to get acquainted with the entire list of commands and capabilities?
I want to write a script, but I don’t even know where to start.
What are the commands, properties, methods, what do they do, what do they return?

**Comment by skelgaard** [comment]
the easiest way is to come to the discord and talk to them in the scripts channel... here are some ppl that have done some scripts :)
